src/common/src/config.rs

```diff
@@ -1130,6 +1127,10 @@ pub mod default {
     }
 
     pub mod auto_dump_heap_profile {
+        pub fn enabled() -> bool {
+            true
```

Comment:
Where will we read the env var `MALLOC_CONF`? 👀

Reply:
Here: https://github.com/risingwavelabs/risingwave/pull/12186/files#diff-22db760b18e3eab5a13a58cf0ef5249aa2defa83c011f73bf72e41e58bb8363bR131-R132

```
let prof_prefix_mib = jemalloc_prof::prefix::mib().unwrap();
let prof_prefix = prof_prefix_mib.read().unwrap();
```

It's actually read from `jemalloc` lib as its option.